include/beman/optional26/detail/iterator.hpp

```diff
@@ -43,18 +43,20 @@ struct contiguous_iterator : public base_contiguous_iterator<T, Container> {
     using typename base_type::reference;
 
     // Default constructor.
-    contiguous_iterator() noexcept : m_current() {}
+    constexpr contiguous_iterator() noexcept : m_current() {}
```

Comment:
I also think we should revisit the test suite from https://github.com/beman-project/optional26/blob/main/src/beman/optional26/tests/detail/iterator.t.cpp and add some tests for the constexpr usage.

Reply:
@frederick-vs-ja , also add one more entry into examples/

Reply:
The coverage for constexpr would be somehow large, I'll add it later today.

Reply:
Added coverage in `optional_range_support.t.cpp` and `iterator.t.cpp`, but some stuffs don't work with GCC11. I'll check this soon.